src/lib/ctable.lua

```diff
@@ -219,9 +221,19 @@ function CTable:insert(hash, key, value, updates_allowed)
 
    local entries = self.entries
    local scale = self.scale
-   local start_index = hash_to_index(hash, self.scale)
+   -- local start_index = hash_to_index(hash, self.scale)
```

Comment:
I left it there so as to be greppable in case the function needs changing -- there are a few more instances of the function call in the file and I thought it wasn't so great to inline them all; dunno.